src/main/java/seedu/taassist/storage/JsonAdaptedStudent.java

```diff
@@ -8,19 +8,19 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import seedu.taassist.commons.exceptions.IllegalValueException;
-import seedu.taassist.model.moduleclass.StudentModuleData;
 import seedu.taassist.model.student.Address;
 import seedu.taassist.model.student.Email;
 import seedu.taassist.model.student.Name;
 import seedu.taassist.model.student.Phone;
 import seedu.taassist.model.student.Student;
+import seedu.taassist.model.student.StudentModuleData;
 
 /**
  * Json-friendly version of {@link Student}.
  */
 class JsonAdaptedStudent {
 
-    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Student's %s field is missing!";
+    public static final String MESSAGE_MISSING_FIELD_FORMAT = "Some student's %s field is missing!";
```

Comment:
Perhaps just make this `Student's %s field is missing`? to make it consistent with other Storage class's responses.